registry/api/v2/errors.go

```diff
@@ -40,16 +40,6 @@ var (
 		HTTPStatusCode: http.StatusBadRequest,
 	})
 
-	// ErrorCodeSizeInvalid is returned when uploading a blob if the provided
-	ErrorCodeSizeInvalid = errcode.Register(errGroup, errcode.ErrorDescriptor{
```

Comment:
We should take it through a separate PR, since it was part of the specification.